setup.sh

```diff
@@ -19,6 +19,10 @@ if [[ $STRICT_SCRIPT =~ $TRUE_REG ]]; then
     set -o pipefail
 fi
 
+source /build.info
+export LT_PACKAGE="${LT_PACKAGE:=docker}"
```

Comment:
`=` → `-`
```export LT_PACKAGE="${LT_PACKAGE:=docker}"```
To
```export LT_PACKAGE="${LT_PACKAGE:-docker}"```

Reply:
Also another comment, usually we avoid quotes when settings variables this way. Again, for the sake of uniformity. We should have a bash guide somewhere. :)